Add Avosdim brand and details in the motionblinds documentation (#45664)
home-assistant/home-assistant.github.io: source/_integrations/motion_blinds.markdown

  3. Tap the screen 5 times while being on the **About** page. 
      - This opens a window with the API key.

### Avosdim app

> In the the Avosdim mobile app, go to settings (three bars in the upper left corner > gear icon), select `About` from the bottom, quickly tap the connected shutter icon in the center of the screen 5 times and a pop-up with the key will be shown.

Reviewer: ```suggestion
In the Avosdim mobile app, go to **Settings** (three bars in the upper-left corner > gear icon), select **About** at the bottom, quickly tap the connected shutter icon in the center of the screen 5 times, and a pop-up with the key will appear.
```
